apis/core/v1beta1/common/common.go

```diff
@@ -12,6 +16,38 @@ const (
 	SyncProviderFlagdProxy SyncProviderType = "flagd-proxy"
 )
 
+const (
+	ManagementPortEnvVar        string = "MANAGEMENT_PORT"
+	PortEnvVar                  string = "PORT"
+	HostEnvVar                  string = "HOST"
+	TLSEnvVar                   string = "TLS"
+	SocketPathEnvVar            string = "SOCKET_PATH"
+	OfflineFlagSourcePathEnvVar string = "OFFLINE_FLAG_SOURCE_PATH"
+	SelectorEnvVar              string = "SOURCE_SELECTOR"
+	CacheEnvVar                 string = "CACHE"
+	CacheMaxSizeEnvVar          string = "MAX_CACHE_SIZE"
+	ResolverEnvVar              string = "RESOLVER"
+	EvaluatorEnvVar             string = "EVALUATOR"
+	ImageEnvVar                 string = "IMAGE"
+	VersionEnvVar               string = "TAG"
+	ProviderArgsEnvVar          string = "PROVIDER_ARGS"
+	DefaultSyncProviderEnvVar   string = "SYNC_PROVIDER"
+	LogFormatEnvVar             string = "LOG_FORMAT"
+	ProbesEnabledVar            string = "PROBES_ENABLED"
+	DefaultEnvVarPrefix         string = "FLAGD"
+	DefaultManagementPort       int32  = 8014
+	DefaultPort                 int32  = 8013
```

Comment:
proposal - introrudce a new port for gRPC sync [1] named `defaultGrpcSyncPort` with value 8015 and use that for `FeatureFlagInProcessConfiguration` CRD

[1] - https://flagd.dev/reference/grpc-sync-service/

Reply:
I think this is a good idea, maybe requirement. Good point.

Reply:
ok, do I understand correctly that the default value of the port (currently 8013) should be changes to 8015? (only for in-process) Or do we expect an additional parameter here with 8015 as default value?

Reply:
I think basically, 8013 should be the default if we are using rpc, and 8015 should be that default if we are using in-process.

Cc @beeme1mr @Kavindu-Dodan I think we should do this for the provider defaults too outside of OFO... I think it will be a common misstep for people to change the resolver to in-process and fail to update the port to 8015. I've made this mistake myself.